model/src/main/proto/interaction_object.proto

```diff
@@ -49,3 +50,19 @@ message Fraction {
   int32 denominator = 4;
 }
 
+// Structure for a ListOfString object.
+message ListOfSetsOfHtmlStrings {
+  repeated StringList set_of_html_strings = 1;
+}
+
+// Structure for a DragAndDropHtmlString object.
+message DragAndDropHtmlString {
+  string html = 1;
+}
+
+// Structure for a DragAndDropPositiveInt object.
```

Comment:
While I know this & the other new doc strings you added are just following existing patterns, can we maybe make this more useful? I'm thinking a link to the backend structure might be nice for reference.

Reply:
We can just add a single link at the top of the file as all the models are declared in a single file

Reply:
That sounds good.